sst/nights/facilities_temperatures_reports.ipynb

```diff
@@ -0,0 +1,442 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #68.    heights = {
</pre><p>Are these heights being used? And then the actual heights in the plots seem a bit arbitrary to me. I don't know why Standby should be on top for instance. Makes more sense for the heights to be the ones you have here but they are not used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/reports-performance-summary/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>They were at some point. I removed them.</p>